apps/src/p5lab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -263,7 +254,7 @@ export default class AnimationPickerBody extends React.Component {
                 </div>
               )}
             {((searchQuery === '' && categoryQuery === '') ||
-              (results.length === 0 && this.props.canDraw)) && (
```

Comment:
Why can we remove the check on `canDraw`?

Reply:
`canDraw` was used to keep the "draw your own" out of the library when browsing the backgrounds category. It was there because at some point we didn't want students to be able to draw their own backgrounds in Sprite Lab.